refl1d/view/data_view.py

```diff
@@ -1,25 +1,19 @@
-from __future__ import with_statement
-
+import matplotlib.pyplot as plt
 import wx
-
-# Can't seem to detect when notebook should be drawn on Mac
-IS_MAC = wx.Platform == "__WXMAC__"
-
-from numpy import inf
-
+from bumps.fitproblem import FitProblem
+from bumps.gui.util import EmbeddedPylab
 from matplotlib.backends.backend_wxagg import FigureCanvasWxAgg as FigureCanvas
 from matplotlib.backends.backend_wxagg import NavigationToolbar2WxAgg as Toolbar
 
 # The Figure object is used to create backend-independent plot representations.
 from matplotlib.figure import Figure
-
-import matplotlib.pyplot as plt
```

Comment:
This is probably fine, and we aren't really supporting wx gui anymore, but the order of import statements might matter, if the backend is being specified in one of the imports.

Reply:
Unfortunately I was never able to get wx installed because of lack of python version support, so I can't really test this. Can revert it if you prefer

Reply:
I think Paul approved complete deprecation of the WX gui in version 1.0, so it's up to you.  Not much point in fixing the formatting of the code, but not much harm either.

Removal of the `view` folder will take a bit of work - there are pieces that get used in `export results`, for instance, that have to be excised first.

Reply:
cool ok, may as well leave it then, and work on just removing `view/` later on. i don't expect any further changes to files here formatting wise